### PR TITLE
Feat update localflow

### PR DESCRIPTION
**Pull request by tdivoll**
This updates the local flow.cylc to work from the Docker containers, and hopefully alleviates the ease of use for running locally on simple examples, particularly for fetching data from NASA Worldview and Space-Track APIs.

### CODE REVIEW

**Review by cpaniaguam — CHANGES_REQUESTED** [review]
This is good! Added some comments for readability.

File: README.md [hunk 1]
```diff
@@ -4,10 +4,35 @@ This repository contains the processing pipeline for IceFloeTracker.jl and ancil
...
 **Note:** The `pass_time_cylc.py` script in this project can be adapted to include additional satellites available in the [space-track.org](https://www.space-track.org/) repository.
 
+## Fetching data from NASA Worldview
+
+To use the `fetchdata.sh` script in this repository, all the software dependencies are found in the Docker container at `docker://brownccv/icefloetracker-fetchdata:main`.
+
+To fetch data independently of other tasks:
+1. Make sure Docker Desktop is running on your local machine.
+2. Update inputs and run the following: `docker run --mount type=bind,source=<your_desired_output_dir>,target=/tmp brownccv/icefloetracker-fetchdata:main /usr/local/bin/fetchdata.sh -o /tmp -s <YYYY-MM-DD> -e <YYYY-MM-DD> -c <wgs84|epsg3413> -b <top_left_lat@top_left_lon@lower_right_lat@lower_right_lon|left_x@top_y@right_x@lower_y`
+   * be sure to replace source, s(startdate), e(enddate), c(crs), and b(bounding box) with your inputs
```
[the remark this answers is not in the document]
Comment (tdivoll) [reply README.md:31]: Fetchdata only has short flags; longer names are used in the script to create the necessary variables so the long names are reserved for that purpose.

File: config/cylc_local/flow.cylc [hunk 2]
```diff
@@ -1,74 +1,132 @@
...
             reflectance_dir = $fetchdata_dir/"reflectance"
-            landmask_dir = $results_dir/"landmasks"
-            preprocess_dir = $results_dir/"preprocess"
-            soit_dir = $results_dir/"soit"
-            tracker_dir = $results_dir/"tracker"
-            h5_dir = $preprocess_dir/"hdf5-files"
 
-    [[mkpaths<startdate>]]
+    [[mkpaths<param_set>]]
+        # Convert param location list to indexable bash array and make directories for each location
```
[the remark this answers is not in the document]
Comment (tdivoll) [reply config/cylc_local/flow.cylc:37]: `sed` is just tokenizing the string inputs, for as many inputs as there are in the list, which we pipe into a bash array. I added `# sed is used to tokenize the list of comma-separated inputs` to compliment the original comment `# Convert param location list to indexable bash array and make directories for each location`

File: README.md [hunk 3]
```diff
@@ -15,23 +15,23 @@ To run SOIT manually :
         * when exporting the following environment variables, there __must__ be a space in front of each command
    - [ ] ` export SPACEUSER=<firstname>_<lastname>@brown.edu`
    - [ ] ` export SPACEPSWD=<password>`
-3. Update inputs and run the following: `docker run --env SPACEUSER --env SPACEPSWD --mount type=bind,source=<your_desired_output_dir>,target=/tmp brownccv/icefloetracker-fetchdata:main python3 /usr/local/bin/pass_time_cylc.py --startdate <YYYY-MM-DD> --enddate <YYYY-MM-DD> --csvoutpath /tmp --centroid_x <input_centroid_x> --centroid_y $<input_centroid_y> --SPACEUSER $SPACEUSER --SPACEPSWD $SPACEPSWD`
+3. Update inputs and run the following: ```docker run --env SPACEUSER --env SPACEPSWD --mount type=bind,source=<your_desired_output_dir>,target=/tmp brownccv/icefloetracker-fetchdata:main python3 /usr/local/bin/pass_time_cylc.py --startdate <YYYY-MM-DD> --enddate <YYYY-MM-DD> --csvoutpath /tmp --centroid_x <input_centroid_x> --centroid_y $<input_centroid_y> --SPACEUSER $SPACEUSER --SPACEPSWD $SPACEPSWD```
```
Comment (cpaniaguam) [inline README.md:18]:
```suggestion
3. Update inputs and run the following:
```
docker run --env SPACEUSER --env SPACEPSWD --mount type=bind,source=<your_desired_output_dir>,target=/tmp brownccv/icefloetracker-fetchdata:main python3 /usr/local/bin/pass_time_cylc.py --startdate <YYYY-MM-DD> --enddate <YYYY-MM-DD> --csvoutpath /tmp --centroid_x <input_centroid_x> --centroid_y $<input_centroid_y> --SPACEUSER $SPACEUSER --SPACEPSWD $SPACEPSWD
```

Comment (cpaniaguam) [reply README.md:18]: The change is not being rendered properly. Each set of triple back ticks should be in a line by itself so the formatting is applied correctly. Like this:
'''
comand # use back ticks instead
'''

**Pull request merged**